src/shell/scripts/omp.zsh

```diff
@@ -171,8 +171,6 @@ posh::call_widget()
 # $1: The name of the widget to decorate
 # $2: The name of the posh function to decorate it with
 function posh::decorate_widget() {
-  typeset -F SECONDS
-  local prefix=orig-s$SECONDS-r$RANDOM # unique
```

Comment:
IMO, a system-wide unique identifier is not even required here, since they don't work across sessions. A namespace-like one, e.g., `_omp::original::`, is good enough.

Reply:
It was only in case someone sources it several times. I was probably too cautious.